gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcReadChannel.java

```diff
@@ -85,10 +86,18 @@
   private final BackOffFactory backOffFactory;
 
   // Context of the request that returned resIterator.
-  @Nullable CancellableContext requestContext;
+  @Nullable
+  CancellableContext requestContext;
 
   Fadvise readStrategy;
 
+  @Nullable
+  private final ByteString footerContent;
+
+  private final long footerStartOffset;
+
+  private final long footerSize;
```

Comment:
Do we need to store this value? Given that you can always to footerContent.size()?